ApplicationLibCode/ProjectDataModel/RimProject.cpp

```diff
@@ -1433,6 +1428,16 @@ void RimProject::defineUiTreeOrdering( caf::PdmUiTreeOrdering& uiTreeOrdering, Q
     {
         uiTreeOrdering.add( m_plotTemplateTopFolder );
     }
+    else if ( uiConfigName == "PlotWindow.Cloud" )
+    {
+        if ( m_mainPlotCollection )
+        {
+            if ( activeOilField()->cloudDataCollection() )
+            {
+                uiTreeOrdering.add( activeOilField()->cloudDataCollection() );
+            }
+        }
+    }
```

Comment:
Should the new Cloud docking window be added to the default docking layout? If so, the RiuDockWidgetTools::defaultPlotDockState() method needs to be updated.

Reply:
Fixed.